### DISCUSSION TITLE
Cannot unpin a show and tell discussion?

### DISCUSSION THREAD

**Discussion by slowrunner** [turtlebot/turtlebot4#331] [Q&A] [answered]
I originally posted a “show and tell” in the discussion section and “pinned” it.  After I closed it, I cannot unpin it.  Any idea how I can unpin it?

https://github.com/turtlebot/turtlebot4/discussions/81#discussion-4603171

**Comment by hilary-luo** [comment, marked as answer]
I unpinned it, you should be able to manage it in the same place you would have originally pinned it (bottom right)

**Reply by slowrunner** [reply]
That was where I was looking, but only Transfer and Create Issue options were available.  Thanks for unpinning it - that did the trick.
